packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -267,7 +251,45 @@ export const Dismissible: StoryObj<ToastProps> = {
     linkUrl: "#",
   },
   play: async ({ args, canvasElement, step }) => {
-    openToast({ canvasElement, step })(args, "Dismissible Toast");
+    const canvas = within(canvasElement);
+    await step(`open Dismissible Toast}`, async () => {
+      await waitFor(() => {
+        const buttonElement = canvas.getByText(`Open ${args.severity} toast`);
+        userEvent.hover(buttonElement);
+        userEvent.click(buttonElement);
+        userEvent.tab();
+      });
+    });
+
+    await step("link in toast", async () => {
+      await waitFor(() => {
+        const toastLink = canvas.getByText(args.linkText || "");
+        expect(toastLink).toHaveAttribute("href", args.linkUrl);
+      });
+    });
+
+    await step("dismiss toast and reopen", async () => {
+      await waitFor(() => {
+        const toastElement = canvas.getByRole("status");
+        if (toastElement) {
+          const dismissToastButton = toastElement.querySelector(
+            '[aria-label="close"]',
+          );
+          if (dismissToastButton) {
+            userEvent.click(dismissToastButton);
+            waitFor(() => {
+              expect(toastElement).not.toBeInTheDocument();
```

Comment:
This is our intention right?
```suggestion
              expect(toastElement).not.toBeVisible();
```

Or do we care about it's existence in the DOM over its visibility?

Reply:
After removed the toast, the code was removed from the DOM. So, it should be good to use `toBeInTheDocument`.

Reply:
I agree that `toBeInTheDocument` can work here too.

Although, I think that `toBeVisible` is better to be consistent with the rest of the code. It also checks if it's visible for screen readers.

It's just more clear on our expectations: "Can a user see this?" versus "can the computer see it?".